aws_lambda_builders/workflows/nodejs_npm_esbuild/actions.py

```diff
@@ -27,20 +29,20 @@ class EsbuildBundleAction(BaseAction):
 
     def __init__(
         self,
-        scratch_dir: str,
-        artifacts_dir: str,
+        working_directory: str,
+        output_directory: str,
         bundler_config: Dict[str, Any],
         osutils: OSUtils,
         subprocess_esbuild: SubprocessEsbuild,
         manifest: str,
         skip_deps=False,
     ):
         """
-        :type scratch_dir: str
-        :param scratch_dir: an existing (writable) directory for temporary files
+        :type working_directory: str
```

Comment:
nit: can we also update doc formats to numpy format that you added above (https://www.dataquest.io/blog/documenting-in-python-with-docstrings/)